### PR TITLE
feat(web): allow to limit number of redis connections

### PR DESCRIPTION
**Pull request by sileht**
The default pool limit is 2**31, this allows to configure it.

### CODE REVIEW

File: mergify_engine/config.py [hunk 1]
```diff
@@ -114,6 +114,12 @@ def AccountTokens(v: str) -> typing.Dict[str, str]:
         voluptuous.Required(
             "REDIS_SSL_VERIFY_MODE_CERT_NONE", default=False
         ): CoercedBool,
+        voluptuous.Required(
+            "REDIS_STREAM_WEB_MAX_CONNETIONS", default=None
```
Comment (jd) [inline mergify_engine/config.py:118]:
```suggestion
            "REDIS_STREAM_WEB_MAX_CONNECTIONS", default=None
```

File: mergify_engine/config.py [hunk 2]
```diff
@@ -114,6 +114,12 @@ def AccountTokens(v: str) -> typing.Dict[str, str]:
...
+        ): voluptuous.Any(None, voluptuous.Coerce(int)),
+        voluptuous.Required(
+            "REDIS_CACHE_WEB_MAX_CONNETIONS", default=None
```
Comment (jd) [inline mergify_engine/config.py:121]:
```suggestion
            "REDIS_CACHE_WEB_MAX_CONNECTIONS", default=None
```

File: mergify_engine/config.py [hunk 3]
```diff
@@ -177,6 +183,8 @@ def AccountTokens(v: str) -> typing.Dict[str, str]:
 CONTEXT: str
 ACCOUNT_TOKENS: typing.Dict[str, str]
 REDIS_SSL_VERIFY_MODE_CERT_NONE: bool
+REDIS_STREAM_WEB_MAX_CONNETIONS: typing.Optional[int]
+REDIS_CACHE_WEB_MAX_CONNETIONS: typing.Optional[int]
```
Comment (jd) [inline mergify_engine/config.py:187]:
```suggestion
REDIS_STREAM_WEB_MAX_CONNECTIONS: typing.Optional[int]
REDIS_CACHE_WEB_MAX_CONNECTIONS: typing.Optional[int]
```

**Review by jd — CHANGES_REQUESTED** [review]

**Comment by sileht** [comment]
Wierd mypy didn't catch them

File: mergify_engine/web/__init__.py [hunk 4]
```diff
@@ -50,8 +50,12 @@
 @app.on_event("startup")
 async def startup():
```
Comment (jd) [inline mergify_engine/web/__init__.py:51]: This is not typed, this is why mypy did not catch it.
Would you mind typing it?

**Review by jd — CHANGES_REQUESTED** [review]

File: mergify_engine/web/__init__.py (hunk 4 above)
Comment (sileht) [reply mergify_engine/web/__init__.py:51]: Ah thx!

**Review by jd — APPROVED** [review]

**Pull request merged**